--- config/rules.mk~	Thu Oct 11 23:40:31 2001
+++ config/rules.mk	Mon Aug  5 19:40:06 2002
@@ -39,18 +39,18 @@
 	-cvs2cl.pl
 
 installdirs:
-	mkdir -p $(bindir)
-	mkdir -p $(libgamesdir)
-	mkdir -p $(datagamesdir)
-	mkdir -p $(datadir)/pixmaps
+	mkdir -p $(DESTDIR)$(bindir)
+	mkdir -p $(DESTDIR)$(libgamesdir)
+	mkdir -p $(DESTDIR)$(datagamesdir)
+	mkdir -p $(DESTDIR)$(datadir)/pixmaps
 
 install: installdirs $(TARGETS)
-	$(INSTALL_PROGRAM) quadra $(bindir)/quadra
+	$(INSTALL_PROGRAM) quadra $(DESTDIR)$(bindir)/quadra
 ifdef UGS_LINUX_SVGA
-	$(INSTALL_PROGRAM) quadra-svga.so $(libgamesdir)/quadra-svga.so
+	$(INSTALL_PROGRAM) quadra-svga.so $(DESTDIR)$(libgamesdir)/quadra-svga.so
 endif
-	$(INSTALL_DATA) quadra.res $(datagamesdir)/quadra.res
-	$(INSTALL_DATA) images/quadra.xpm $(datadir)/pixmaps/quadra.xpm
+	$(INSTALL_DATA) quadra.res $(DESTDIR)$(datagamesdir)/quadra.res
+	$(INSTALL_DATA) images/quadra.xpm $(DESTDIR)$(datadir)/pixmaps/quadra.xpm
 # FIXME: the Quadra.desktop file should go to these places:
 # /etc/X11/applnk/Games/Quadra.desktop
 # /usr/share/gnome/apps/Games/Quadra.desktop
